packages/eslint-plugin/src/rules/member-ordering.ts

```diff
@@ -934,6 +967,17 @@ export default createRule<Options, MessageIds>({
       let memberTypes: MemberType[] | string | undefined;
       let optionalityOrder: OptionalityOrder | undefined;
 
+      const runSort = (memberSet: Member[]): undefined => {
```

Comment:
[Style] Similarly, this name kind of confused me. Everything here can arguably be referred to as "run(ning a) sort". Maybe ... `checkAlphaSortForAllMembers`? `runSortForAlphaGroup`? What would you say this function is really trying to do?